workers/generic-worker/main.go

```diff
@@ -691,6 +696,20 @@ func (task *TaskRun) validateJSON(input []byte, schema string) *CommandExecution
 	return MalformedPayloadError(fmt.Errorf("Validation of payload failed for task %v", task.TaskID))
 }
 
+func validateGenericWorkerBinary() error {
+	cmd, err := gwVersion()
+	if err != nil {
+		return fmt.Errorf("could not determine generic-worker binary version: %v", err)
```

Comment:
I think the error here means you couldn't generate the command to determine what the generic-worker binary is, doesn't it? I think at this point the command hasn't been executed, so this should be a panic if we can't even create the command, I think?